src/interfaces/ffi/tests.rs

```diff
@@ -239,9 +240,9 @@ unsafe fn encrypt_header_using_cache(
     header_metadata: &[u8],
     authentication_data: &[u8],
 ) -> Result<(<DEM as Dem<{ DEM::KEY_LENGTH }>>::Key, EncryptedHeader), Error> {
-    let policy_cs = CString::new(serde_json::to_string(&policy)?.as_str())
-        .map_err(|e| Error::Other(e.to_string()))?;
-    let policy_ptr = policy_cs.as_ptr();
+    let policy_bytes = serde_json::to_vec(&policy)?;
```

Comment:
`serde_json::to_vec` is just JSON then UTF8 bytes? I prefer to have it explicitly in two steps than hidding it in `serve_json::to_vec`

Reply:
will be done in next release